cmd/utils/utils.go

```diff
@@ -9,12 +9,8 @@ import (
 )
 
 const (
-	LatestVersion = "latest"
-	//nolint: gosec // this is not a secret leak, it's just a format declaration.
-	DefaultCoreDockerImage    = "ghcr.io/calyptia/core"
-	DefaultCoreDockerImageTag = LatestVersion
-
-	DefaultCoreOperatorDockerImage = "ghcr.io/calyptia/core-operator"
+	LatestVersion                  = "latest"
+	DefaultCoreOperatorDockerImage = "ghcr.io/chronoshpereio/core-operator"
```

Comment:
No, let's keep the current registry until we have properly covered migration in docs/tests/etc.
Plus this has a typo

Reply:
done